openreview/tools.py

```diff
@@ -1577,6 +1577,7 @@ def get_profile_info(profile):
     emails = set()
     relations = set()
     publications = set()
+    publication_years = 3
```

Comment:
do you think the default should be last 3 years? I was thinking to use all the existing publications as the default implementation.

Reply:
Depends on how much extra time would it take.

Reply:
It shouldn't take too much time to compare the author relationship. Then we can parametrize the last year for publications,  history of domains and relations.

Reply:
I have made the default to consider all publications.